lib/car/car.worker.ts

```
@@ -16,35 +19,79 @@ import { Edge } from "@/lib/primitives/edge";
 import { Node } from "@/lib/primitives/node";
 import { Polygon } from "@/lib/primitives/polygon";
 import { EdgeJson, PolygonJson } from "@/types/save";
+import { NeuralNetwork } from "@/lib/ai/network";
+import { ControlType } from "@/lib/car/controls";
 
 let carState: WorkerCarState;
 
 onmessage = (event: MessageEvent<CarWorkerInboundMessage>) => {
```

Biome (2.4.4)

[error] A global variable should not be reassigned.

(lint/suspicious/noGlobalAssign)